### ISSUE TITLE
PNI - Product Review - Tabs not accessible for certain locales

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#10101]
### Describe the bug
PNI Product Review Page - Tabs are not accessible for certain locales because the tab labels (controls) are either cut off or not visible within viewport.

### To Reproduce
1. Go to any product review page on PNI, e.g., https://foundation.mozilla.org/privacynotincluded/signal/
2. Shrink the browser size so it's < 992px wide
3. Select Polish as locale
4. Scroll down to the tab group
5. Notice tab group text is cut off
6. Depending on the screen size, sometimes you won't be able to go back to the first tab since the first tab label is out of viewport. (Example flow: on 702px wide screen, click on second tab label "Bezpieczeństwo", now you can't get to the first tab anymore since the tab label is out of the viewport)

### Expected behavior
Tab controls (labels) are visible and accessible for all screen sizes and locales.

### Screenshots

https://user-images.githubusercontent.com/2896608/216434198-507dacb3-7663-4d62-8340-bbfa65c9cde6.mov

**Comment by mmmavis** [comment]
We'll need design support to find a solution. @kristinashu could you direct me to the designer I should work with? Thank you!

**Comment by kristinashu** [comment]
I believe @nancyt1 has previously worked on these tabs. I will put this into our next sprint!

**Comment by mtdenton** [comment]
Reviewed in grooming, we'll need @nancyt1 to weigh in here in order to proceed. Thanks in advance!

**Comment by mmmavis** [comment]
@nancyt1 curious how we want to approach this ticket? Is there a fix you have in mind already?

**Comment by mmmavis** [comment]
Discussed with @nancyt1 . We are gonna try making the tabs horizontally scrollable first and decide on further treatments.

I will keep Nancy and this ticket updated as I go.
